ckanext/ontario_theme/templates/internal/organization/read.html

```diff
@@ -21,13 +21,37 @@
     (_('Last Modified'), 'metadata_modified desc'),
     (_('Popular'), 'views_recent desc') if g.tracking_enabled else (false, false) ]
   %}
+
+  {% if not sort_by_selected %}
+    {% set this_org = c.page.items[0]['organization']['name'] %}
+    {% set sort_by_selected = 'metadata_modified desc' %}
+  {% endif %}
+
   {% set searchbar_text = gettext('Search %(ministry)s datasets', ministry=ministry) %}
   {% set searchbar_title = searchbar_text + _(':') %}
   {% set form_id = 'organization-datasets-search-form' %}
   <form id="{{ form_id }}" class="search-form" method="get" data-module="select-switch" novalidate>
-  {% snippet 'snippets/search_form.html', searchbar_title=searchbar_title, hidden_searchbar_title=searchbar_text, form_id=form_id, type='dataset', query=q, sorting=sorting, sorting_selected=sort_by_selected, count=page.item_count, facets=facets, placeholder=searchbar_text, show_empty=request.params, fields=fields %}
+    {% snippet 'snippets/search_form.html',
+                searchbar_title=searchbar_title,
+                hidden_searchbar_title=searchbar_text,
+                form_id=form_id,
+                type='dataset', 
+                query=q, 
+                sorting=sorting, 
+                sorting_selected=sort_by_selected, 
+                count=page.item_count, 
+                facets=facets, 
+                placeholder=searchbar_text, 
+                show_empty=request.params, 
+                fields=fields %}
   </form>
 {% endblock %}
+{% block packages_list %}
+    {% if page.items %}
```

Comment:
I think the Error [500] is due to no else in this if statement when there is no packages.